frontend/src/features/leases/detail/LeasePages/payment/styles.tsx

```diff
@@ -58,6 +59,61 @@ export const StyledTable = styled(Table)`
         }
       }
     }
+    > .tbody > .tr-wrapper > tr > .td:nth-of-type(2) {
+      margin-right: -5rem; /** TODO: prevent expander from pushing tds to the right */
```

Comment:
This is a mess, but isn't trivial to fix given that the current logic just adds a td to the existing table with the expander icon in it. This pushes all of the tds in the table to the right. Will try to address this sprint or log tech debt for a better solution for future expandable tables.